### PR TITLE
Added question in CfU- JavaScript6

### PR DESCRIPTION
**Pull request by raibove**
Fixed Issue #700

### CODE REVIEW

File: javascript/javascript-6-object-literals.md [hunk 1]
```diff
@@ -89,4 +90,10 @@ var loan = {
 - Work through [this interactive lesson on JavaScript Objects](https://www.codecademy.com/courses/introduction-to-javascript/lessons/objects/exercises/objects?action=resume_content_item).
 
 ### Check for Understanding
-Explain your completed code from the above Codecademy lesson to a partner.
+
+- Write down the syntax for declaring an Object.
+- What do you mean by Complex Objects. Give an example.
```
Comment (Kundan28) [inline javascript/javascript-6-object-literals.md]: Shouldn't there be a question mark ?

Comment (raibove) [reply javascript/javascript-6-object-literals.md]: Do you mean ' What do you mean by Complex Objects? Give an example.'

Comment (Kundan28) [reply javascript/javascript-6-object-literals.md]: Yeah.

Comment (raibove) [reply javascript/javascript-6-object-literals.md]: Ohhk, Got it. I added a question mark

**Pull request merged**